### PR TITLE
v3.10.0

### PR DESCRIPTION
**Pull request by liamcho**

### CODE REVIEW

**Comment by sendbird-sdk-deployment** [comment]
[Creating Ticket] 🔖 Done https://sendbird.atlassian.net/browse/SDKRLSD-1095

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,63 @@
...
+        )}
+      />
+      ```
+  
+### Fixes:
+
+* Fixed a bug in mobile view where channel view is displaying a default channel when there is no channel in channel list
+* Added missing props renderMessageContent in Channel
+* Fixed a bug where center alignment of `Badge` and `Button` components breaking in FireFox browser
+* Fixed a bug where messages sent by bot member in group channel are not triggering the expected hooks
```
Comment (AhyoungRyu) [inline CHANGELOG.md:59]: Please add this line too to thank the original author 
```suggestion
* Fixed a bug where messages sent by bot member in group channel are not triggering the expected hooks (Original Author: [ishubham21](https://github.com/ishubham21))
```

**Comment by liamcho** [comment]
@AhyoungRyu  the commit has been made after the deployment. It will show in next release 🙏

**Review by AhyoungRyu — APPROVED** [review]

**Pull request merged**